base/Dockerfile

```diff
@@ -12,6 +12,8 @@ RUN apt update -y && \
         git \
         libaravis-dev \
         libevent-dev \
+        libnet-dev \
+        libpcap-dev \
```

Comment:
It may be better if you explain in the longer commit message why do you added this packages. (same in alpine Dockerfile)

Reply:
In general, such dependencies are directly related to the libraries used in the source code. So, I usually don't bother to make this explicit. In this case, only a single device type (which we do not currently have) requires such libraries, so I've added a mention to.

Thanks for the question, nevertheless. I noticed we did not install the runtime packages for these two. Attempting to run with a Canberra device would not work.